### PR TITLE
takeaway challenge 

### PR DESCRIPTION
**Pull request by maishachowdhury**
Incomplete

### CODE REVIEW

**Comment by maishachowdhury** [comment]
Incomplete

File: lib/takeaway.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @menu = { "Small Chips" => 2.00,
+             "Medium Chips" => 4.00, 
+             "Large Chips" => 6.00, 
+             "Cod" => 4.50,
+             "Chicken Wings" => 3.50, 
+             "Beans" => 2.00,
+             "Mixed Veg" => 2.40,
+             "Eggs" => 1.25, 
+             "Mushrooms" => 0.70 }
+
```
Comment (FrancineBlanc) [inline lib/takeaway.rb:22]: You could think about moving the menu hash to the `initialize` method, so that when the Takeaway class is initialized, the menu is there as the state of the class

File: lib/takeaway.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+             "Cod" => 4.50,
+             "Chicken Wings" => 3.50, 
+             "Beans" => 2.00,
+             "Mixed Veg" => 2.40,
+             "Eggs" => 1.25, 
+             "Mushrooms" => 0.70 }
+            
+             puts "Please select dishes"
+             puts @menu 
+    end
```
Comment (FrancineBlanc) [inline lib/takeaway.rb:25]: I like that the user is first prompted to make a choice, and is then shown the list of available dishes.

File: lib/takeaway.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+             "Mushrooms" => 0.70 }
+            
+             puts "Please select dishes"
+             puts @menu 
+    end 
+  
+  #Dishes --> hash_of_prices --> select_dish --> dishes_selected 
+  def select_dish(dish)
+    puts "Select dish please"
+    @order.push(dish)
```
Comment (FrancineBlanc) [inline lib/takeaway.rb:30]: You may need some additional code here so that the dish passed in to `select_dish` can be identified from the `menu` hash

File: spec/takeway_spec.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+require "takeaway.rb"
+
+describe Takeaway do 
+  it "shows menu" do
+    expect(subject.dish_price).to eq @menu
+  end
```
Comment (FrancineBlanc) [inline spec/takeway_spec.rb:6]: You may not wish to expose the actual `@menu` variable here

File: lib/confirmation.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+require_relative "takeaway.rb"
+
+class Confirmation
```
Comment (FrancineBlanc) [inline lib/confirmation.rb:3]: Good abstraction of the confirmation class from the takeaway class

File: lib/confirmation.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+require_relative "takeaway.rb"
+
+class Confirmation 
+  #Confirmation --> dishes_selected --> selected_dish_total --> confirmed 
+  #Confirmation --> confirmed --> text_sent 
+  #--> "Thank you! Your order was placed and will be delivered before 18:52"
+  #Method without Twilio API 
+  #Dummy code unfinished 
+
```
Comment (FrancineBlanc) [inline lib/confirmation.rb:9]: Will you be using the Takeaway class as  a dependency here? i.e. initialising `Confirmation` by passing (for example) `takeaway = Takeaway.new` into `initialize` for this class?

File: spec/takeway_spec.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+  it "selects dish" do
+    expect(subject.select_dish("Small Chips")).to eq ["Small Chips"]
```
Comment (FrancineBlanc) [inline spec/takeway_spec.rb:9]: It may be beneficial to test that `subject.order` contains small chips, and move `subject.select_dish("Small Chips") to the line immediately above the `expect`

File: README.md [hunk 8]
```diff
@@ -14,6 +14,53 @@ Takeaway Challenge
...
+
+Confirmation --> dishes_selected --> selected_dish_total --> confirmed 
+
+As a customer
+So that I am reassured that my order will be delivered on time
+I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+
+Confirmation --> confirmed --> text_sent --> "Thank you! Your order was placed and will be delivered before 18:52"
+
+Steps in plain English
```
Comment (FrancineBlanc) [inline README.md:53]: You have given a lot of thought to these user stories and what may be needed to make this program run.
